functions/onprem/orborus/orborus.go

```diff
@@ -2164,32 +2164,22 @@ func deployTenzirNode() error {
 	if err != nil {
 		if strings.Contains(fmt.Sprintf("%s", err), "Conflict. The container name ") {
 			log.Printf("[DEBUG] Tenzir Node Container already exists, starting it")
-
-			err = dockercli.ContainerStart(ctx, containerName, containerStartOptions)
-			if err != nil {
-				log.Printf("[ERROR] Failed to start existing Tenzir Node container: %v", err)
-				return err
-			}
-
-			log.Printf("[INFO] Existing Tenzir Node container started successfully")
-			return nil
+		} else {
+			log.Printf("[ERROR] Failed to create Tenzir container: %s", err)
+			return err
 		}
-		return err
 	}
 
-	log.Printf("[INFO] New Tenzir Node container created successfully")
-
 	err = dockercli.ContainerStart(ctx, containerName, containerStartOptions)
 	if err != nil {
-		log.Printf("[ERROR] Failed to start new Tenzir Node container: %v", err)
+		log.Printf("[ERROR] Failed to start Tenzir Node container: %v", err)
 		return err
 	}
-
-	log.Printf("[INFO] New Tenzir Node container started successfully")
+	log.Printf("[INFO] Tenzir Node container started successfully")
 
 	log.Printf("[INFO] Waiting for tenzir to become available ...")
 	err = checkTenzirNode()
```

Comment:
Is this the only place we do this? Doesn't that mean we will always force start the container before connecting to it? What happens for subsequent requests?